iptables/iptables.go

```diff
@@ -220,7 +231,7 @@ func makeMultiportDestinations(portsToIgnore []string) [][]string {
 	return append(destinationSlices, destinations)
 }
 
-func executeCommand(firewallConfiguration FirewallConfiguration, cmd *exec.Cmd) error {
+func executeCommand(firewallConfiguration FirewallConfiguration, cmd *exec.Cmd, cmdOut io.Writer) error {
```

Comment:
Why is this change needed?

Reply:
We need a way to capture the output of the current iptable rules in the pod in order to detect if they already exist. So we send pass in a buffer into string buffer which satifies the `io.Writer` interface into `executeCommand` to get that and then the caller can decide what it wants to do with that captured output.